src/ProjectTemplates/test/BlazorServerTemplateTest.cs

```diff
@@ -24,18 +25,18 @@ public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory)
     public override string ProjectType { get; } = "blazorserver";
 
     [Fact]
-    public Task BlazorServerTemplateWorks_NoAuth() => CreateBuildPublishAsync("blazorservernoauth");
+    public Task BlazorServerTemplateWorks_NoAuth() => CreateBuildPublishAsync();
 
     [Fact]
-    public Task BlazorServerTemplateWorks_ProgamMainNoAuth() => CreateBuildPublishAsync("blazorservernoauth", args: new[] { ArgConstants.UseProgramMain });
+    public Task BlazorServerTemplateWorks_ProgamMainNoAuth() => CreateBuildPublishAsync(args: new[] { ArgConstants.UseProgramMain });
 
     [Theory]
-    [InlineData(true, null)]
-    [InlineData(true, new string[] { ArgConstants.UseProgramMain })]
-    [InlineData(false, null)]
-    [InlineData(false, new string[] { ArgConstants.UseProgramMain })]
+    [InlineData("Individual", null)]
+    [InlineData("Individual", new string[] { ArgConstants.UseLocalDb })]
+    [InlineData("Individual", new string[] { ArgConstants.UseProgramMain })]
+    [InlineData("Individual", new string[] { ArgConstants.UseProgramMain, ArgConstants.UseLocalDb })]
```

Comment:
nit, could drop the "Individual" and just hardcode that in CreateBuildPublishAsync

Reply:
The auth is passed through from the other theory though too, `BlazorServerTemplate_IdentityWeb_BuildAndPublish`

Reply:
Sorry I just meant drop it from the inline data, and call `CreateBuildPublishAsync("Individual", args: args);` or am I misreading this

Reply:
Yeah doesn't work: 
<img width="842" alt="image" src="https://user-images.githubusercontent.com/249088/167743478-89d402b6-1cb6-4818-b1e2-67fdcca53189.png">

Reply:
Weird oh well